### PR TITLE
PTX-1907  [torpedo] Torpedo failing to validate host on openshift 4.2…

### PR DESCRIPTION
**Pull request by nareshdesh**
modified code to use 'date'

### CODE REVIEW

**Comment by nareshdesh** [comment]
Following torpedo job is using the code changes made in this request
https://jenkins.portworx.co/job/Torpedo/view/Torpedo%202.2/job/tp-openshift4.2-op-2.2-csi/

File: drivers/node/ssh/ssh.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ func (s *SSH) TestConnection(n node.Node, options node.ConnectionOpts) error {
 	if useSSH() {
 		cmd = "hostname"
 	} else {
-		cmd = "cat /etc/hostname"
+		cmd = "date"
```
Comment (harsh-px) [inline drivers/node/ssh/ssh.go]: this is okay for now but we could just use the same hostname command for both. Once we are done with 2.2, you can send another PR to use `cmd = "hostname"` both both. (i.e remove the if condition)

**Comment by thiguetta** [comment]
@nareshdesh could you please run a test on openshift 3.11 before merging, I remember it has some limited command set

**Comment by harsh-px** [comment]
@nareshdesh as thiago requested, let's run this PR on a openshift 4.2 job an merge after it passes.
